REACT-MODULE-PROJECT/src/components/App/App.jsx

```diff
@@ -0,0 +1,15 @@
+import Bookings from "@/components/Bookings/Bookings.jsx";
+import "./App.scss";
+import React from 'react';
+
+
+const App = () => (
+  <div className="app">
+    <header className="app__header">
+      <h1 className="app__heading">CYF Hotel</h1>
+    </header>
```

Comment:
We have already move this header tag into a separate AppHeader component. Did you pulled all changes in your repository because it looks like you have not updated your repository.